modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -71,6 +71,7 @@ export interface DomParserSettings {
   remove_trailing_brs?: boolean;
   root_name?: string;
   sandbox_iframes?: boolean;
+  sandbox_iframes_exclusions?: Record<string, {}>;
```

Comment:
I think it's better to use an array here and the micro optimize this later maybe use a set instead.